### PR TITLE
M0078DDG-500 Update Platform Dev Guide

### PR DESCRIPTION
**Pull request by thomasponsmicroej**
Update of some typos/wording/missing info in the Platform dev guide (creation, migration)

### CODE REVIEW

File: PlatformDeveloperGuide/platformCreation.rst [hunk 1]
```diff
@@ -331,8 +331,8 @@ The publication of the built Platform to a :ref:`module repository <module_repos
 It can be enabled by setting the ``skip.publish`` property defined in the file ``module.properties`` of 
 the Platform configuration project to ``false``.
 
-The publication is generally kept disabled by default in the project sources since developers use the locally built platform,
-but must be enabled in the Continuous Integration environment.
+The publication is kept disabled by default in the project sources because developers usually use the locally built platform in the workspace.
+In a Continuous Integration environment however, the publication is required.
```
Comment (tdelhomenie-microej) [inline PlatformDeveloperGuide/platformCreation.rst:335]: Shouldn't "however" be at the beginning of the sentence?

File: PlatformDeveloperGuide/platformCreation.rst [hunk 2]
```diff
@@ -425,7 +425,7 @@ The 3 most common integration cases are:
   
   The MicroEJ Platform includes the BSP.
 
-  BSP connection is configured when building MicroEJ Platform (relative locations within the BSP), 
+  BSP connection is configured when building the MicroEJ Platform (relative locations within the BSP),
```
Comment (tdelhomenie-microej) [inline PlatformDeveloperGuide/platformCreation.rst:428]:
```suggestion
  BSP connection is configured when building the Platform (relative locations within the BSP), 
```

File: PlatformDeveloperGuide/platformMigration.rst [hunk 3]
```diff
@@ -33,11 +33,19 @@ The migration steps are:
...
 
+
+.. _platform_migration_repository:
+
 Create an Architecture Repository
 ---------------------------------
 
-The first step is to create an Architecture Repository. 
-The MicroEJ Architecture and MicroEJ Packs are provided in the ``platformArchitecture`` directory of the `fullPackaging` package.
+The first step is to create an Architecture Repository, using the MicroEJ Architecture
```
Comment (tdelhomenie-microej) [inline PlatformDeveloperGuide/platformMigration.rst:42]: using -> containing ?

File: PlatformDeveloperGuide/platformMigration.rst [hunk 4]
```diff
@@ -33,11 +33,19 @@ The migration steps are:
...
+
 Create an Architecture Repository
 ---------------------------------
 
-The first step is to create an Architecture Repository. 
-The MicroEJ Architecture and MicroEJ Packs are provided in the ``platformArchitecture`` directory of the `fullPackaging` package.
+The first step is to create an Architecture Repository, using the MicroEJ Architecture 
+and MicroEJ Packs provided in the ``platformArchitecture`` directory of the `fullPackaging` package.
+
+.. note:: If the Architecture and Packs used by the Platform are already stored on the
```
Comment (tdelhomenie-microej) [inline PlatformDeveloperGuide/platformMigration.rst:45]:
```suggestion
.. note:: If the Architecture and Packs used by the Platform are already stored in the 
```

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

File: PlatformDeveloperGuide/platformCreation.rst (hunk 1 above)
Comment (thomasponsmicroej) [reply PlatformDeveloperGuide/platformCreation.rst]: fixed

File: PlatformDeveloperGuide/platformCreation.rst [hunk 5]
```diff
@@ -300,23 +300,23 @@ Platform Customization
 
 The configuration project (the project which contains the
 :guilabel:`.platform` file) can contain an optional :guilabel:`dropins` folder.
-The contents of this folder will be copied integrally into the final
-Platform. This feature allows to add some additional libraries, tools
+The contents of this folder will be copied in full in the Platform during the Platform build.
```
Comment (tdelhomenie-microej) [inline PlatformDeveloperGuide/platformCreation.rst:303]:
```suggestion
The full content of this folder will be copied in the Platform during the build. 
```

File: PlatformDeveloperGuide/platformCreation.rst [hunk 6]
```diff
@@ -300,23 +300,23 @@ Platform Customization
 
 The configuration project (the project which contains the
 :guilabel:`.platform` file) can contain an optional :guilabel:`dropins` folder.
-The contents of this folder will be copied integrally into the final
-Platform. This feature allows to add some additional libraries, tools
+The contents of this folder will be copied in full in the Platform during the Platform build. 
+This feature allows to add some additional libraries, tools
```
Comment (tdelhomenie-microej) [inline PlatformDeveloperGuide/platformCreation.rst:304]: We should we add that it also allows to overwrites existing files?

Comment (thomasponsmicroej) [reply PlatformDeveloperGuide/platformCreation.rst:304]: yes, maybe, there is a chapter in which it is described but maybe add a note here too could be good

Comment (tdelhomenie-microej) [reply PlatformDeveloperGuide/platformCreation.rst:304]: I was thinking of just changing the sentence to something like: "This feature allows to add or overwrite libraries, tools,
etc. into the Platform."

Comment (thomasponsmicroej) [reply PlatformDeveloperGuide/platformCreation.rst]: changed

**Review by tdelhomenie-microej — APPROVED** [review]

**Comment by tdelhomenie-microej** [comment]
@thomasponsmicroej the build fails : https://readthedocs.com/projects/microej-docs/builds/1120383/

**Comment by thomasponsmicroej** [comment]
> @thomasponsmicroej the build fails : https://readthedocs.com/projects/microej-docs/builds/1120383/

ok, fixed, it passes now

**Review by tdelhomenie-microej — APPROVED** [review]

**Pull request merged**